helm/azuredisk-csi-driver-app/templates/_helpers.tpl

```diff
@@ -11,10 +11,21 @@
 
 {{/* labels for helm resources */}}
 {{- define "azuredisk.labels" -}}
-app.kubernetes.io/instance: "{{ .Release.Name }}"
-app.kubernetes.io/managed-by: "{{ .Release.Service }}"
-app.kubernetes.io/name: "{{ template "azuredisk.name" . }}"
-app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
-helm.sh/chart: {{ include "chart" . | quote }}
-application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
+labels:
```

Comment:
it feels weird to include the `labels` keyword inside the helper, I don't think we do it like that anywhere else. Do we need to put it inside the helper?

Reply:
Yes, you're right. I wanted to open the PR as a draft but noticed it too late 🙈. I wanted to see the diff easier. 

I'll change that for sure 👍🏻. Will let you know when it's finished. Sorry for pinging you too early 😬